### ISSUE TITLE
Error after upgrade Laravel v7.19.1

### ISSUE THREAD

**Issue by garbinmarcelo** [nWidart/laravel-modules#1022]
Hi, after updating the dependencies via composer this error is found in the image.
![image](https://user-images.githubusercontent.com/9335588/87334637-589eab00-c515-11ea-92a5-23a733af7523.png)

**Comment by nWidart** [comment]
Hi,

I don't see anything related to the package here. From what version, to what version are you upgrading?
Just in case, make sure you take a look at the release notes: https://github.com/nWidart/laravel-modules/releases
It documents breaking changes in case you're using something from that.

**Comment by garbinmarcelo** [comment]
right, I'm going to go back to the version of Laravel that was working, it seems that the error occurs when it is loaded from the module's settings.

It could be a Laravel bug then or maybe they changed some method that is breaking the application.
The error appears to be related to the ```$config->get($key, [])```, I didn't quite understand why this error.

![image](https://user-images.githubusercontent.com/9335588/87336317-06ab5480-c518-11ea-8fb6-83fa904851cb.png)

Thanks for listening.

**Comment by nWidart** [comment]
Alright well let us know if you upgrade again with the same issue.

**Comment by garbinmarcelo** [comment]
I went back to version Laravel v7.18.0 and nwidart/laravel-modules v7.1.0.

With these versions the application works normally, but if I update to Laravel v20 (now) it ends up giving the error mentioned earlier.

![image](https://user-images.githubusercontent.com/9335588/87554388-75102400-c68a-11ea-9271-00de35787b10.png)

The error also occurs right after the packages are updated, I believe that when the dump-autoload command is occurring.
The error on the screen is related to this package, as it seems to be unable to load the module configurations (call within the module's serviceprovider).

any suggestions for checking what may be happening?

Thank's

**Comment by nWidart** [comment]
Hm that's odd.
There shouldn't be breaking changes in minor versions.

How do you know it's in one of your module's service providers (I don't see it in the screenshot)?

You can debug it by seeing where it errors and why. Disable all caching for this.
I see things related to cache in that screenshot so clear the cache , config cache etc too.

**Comment by garbinmarcelo** [comment]
I tried to do a test here, I removed all the third party packages that I had installed and left only yours. However, the same error occurred.
In the screenshot you can see if the line where it is written: `app/Modules/Authorization/Providers/AuthorizationServiceProvider.php:83`

so I think it’s something related to the package. Another point I did test is to disable all modules via `modules_statuses.json` and run the composer install again, in this case with all modules disabled the error does not occur.

I tried to activate only 1 different module from which the error was occurring, but the error remains the same.

**Comment by garbinmarcelo** [comment]
Hi,

Ok, I did the same thing here and everything is really normal with the package and Laravel.

I did some other tests and found the problem. In my config files inside the modules I'm using an array with some information from the modules, it always worked that way, but in this last update of Laravel ended up breaking this part, I don't know why.

Less bad, so all right and thanks for your attention and sorry for the inconvenience. I'm finalizing this issue.
